freetype.sh

```diff
@@ -7,7 +7,7 @@ build_requires:
   - system-curl
 prefer_system: (?!slc5)
 prefer_system_check: |
-  printf "#include <ft2build.h>\n" | c++ -xc++ - `freetype-config --cflags` -c -M 2>&1;
+  printf "#include <ft2build.h>\n" | c++ -xc++ - `pkg-config freetype2 --cflags` -c -M 2>&1;
```

Comment:
I would suggest we leave both:

```suggestion
  printf "#include <ft2build.h>\n" | c++ -xc++ - `freetype-config --cflags 2>/dev/null` `pkg-config freetype2 --cflags 2>/dev/null` -c -M 2>&1;
```

since there are platforms (most notably mac) where pkgconfig is not (yet?) a dependency of the freetype you get. Having both should not have any significant side effect.